website/content/docs/enterprise/lts.mdx

```diff
@@ -0,0 +1,195 @@
+standard support for a given Vault version lasts approximately 1 year:
+
+Version | Release date         | Standard support ends
+------- | -------------------- | ----------------------
+1.16    | CY24 Q1 (2024-03-27) | CY25 Q1 (1.19 release) 
+1.15    | CY23 Q3 (2023-09-27) | CY24 Q3 (1.18 release) 
+1.14    | CY23 Q2 (2023-06-21) | CY24 Q2 (1.17 release) 
+1.13    | CY23 Q1 (2023-03-01) | CY24 Q1 (2024-03-27) 
+1.12    | CY22 Q4 (2022-10-12) | CY23 Q3 (2023-09-27) 
+
+**Long-term support** extends support for specific Vault versions for an
+additional year with:
+
+- patches for bugs that may cause outages.
+- patches for critical vulnerabilities and exposures.
+- high-risk security patches.
+
+After the first year, LTS Vault versions move from standard support to long-term
+support for an additional year.
+
+<a id="where" />
+
+## Where do I enable long-term support?
+
+You do not need to download a separate binary or set a flag for LTS support. As
+long as you select an LTS Vault Enterprise version (e.g., 1.16, 1.19) when you
+[install](/vault/install) or [upgrade](/vault/docs/upgrading) your Vault
+instance, LTS is included.
+
+<a id="when" />
+
+## When are LTS versions released?
+
+As of Vault Enterprise 1.16, the first release of a calendar year includes
+long-term support.
+
+LTS versions overlap by one year with the previous LTS version entering its
+long-term support window when the new LTS version begins its standard support
+window.
+
+<a id="why" />
+
+## Why is there a risk to updating to a non-LTS Vault version?
+
+Long-term support is intended for Enterprise customers who cannot upgrade
+every quarter to stay within the standard support timeline of one year. The
+goal is to establish a predictable upgrade path with a longer timeline rather
+than extending the lifetime for every Vault version.
+
+Long-term support ensures your Vault version continues to receive critical
+patches for an additional year. If you upgrade to a non-LTS version, you are
+moving your Vault instance to a version that lacks extended support, which means
+your Vault version stops receiving updates once it leaves the standard support
+window:
+
+Version | Expected release | Standard support ends  | Long-term support ends
+------- | ---------------- | ---------------------- | ---------------------
+1.19    | CY25 Q1          | CY26 Q1 (1.22 release) | CY27 Q1 (1.25 release)
+1.18    | CY24 Q3          | CY25 Q3 (1.21 release) | Not provided
+1.17    | CY24 Q2          | CY25 Q2 (1.20 release) | Not provided
+1.16    | CY24 Q1          | CY25 Q1 (1.19 release) | CY26 Q1 (1.22 release) 
+
+If a newer version of Vault includes features you want to take advantage of, you
+have two options:
+
+1. Wait for the next available LTS release to maintain long-term support.
+1. Upgrade immediately, then upgrade to an LTS release before the standard
+   support window expires.
+
+<a id="how" />
+
+## How do I upgrade my Vault installation?
```

Comment:
I'm confused by this section. The first sentence states I shouldn't change my current behavior, but the next sentence implies that I should be upgrading from LTS to LTS+1 (which may not match my current behavior).

it seems like there's more to explain here?

Reply:
Not really. You don't necessarily need to change your testing and deployment process, but you should allow more time since you're always jumping at least 3 versions between LTS releases. Can you elaborate on what is specifically causing confusion?